spec/enumerable_spec.rb

```diff
@@ -0,0 +1,92 @@
+require './main'
+
+describe Enumerable do
+  describe '#each' do
+    it 'Add 2 to each item' do
+      result = []
+      [1, 2, 3, 4].my_each { |item| result.push(item + 2) }
+      expect(result).to eql([3, 4, 5, 6])
+    end
```

Comment:
Kindly you have not tested for some scenarios. Kindly check below ⬇️ 

- [ ] If no block is given this method should return an enumerator. You should check for this too.

- [ ]  Test for negative scenarios. Remember this is a requirement.  ⬇️ 
![commonsense](https://user-images.githubusercontent.com/32361963/102613487-9dbdad80-4143-11eb-93fc-9aa05749986d.png)